src/Conversionfunctions.jl

```diff
@@ -0,0 +1,23 @@
+module Conversionfunctions
+
+export Taitt
+import Convertible: findpath, haspath
+include("constants.jl")
```

Comment:
If this won't be a module on its own, this include should probably be moved elsewhere.